lib/Raven/Configuration.php

```diff
@@ -803,7 +770,7 @@ private function configureOptions(OptionsResolver $resolver)
                 $this->server .= ':' . $parsed['port'];
             }
 
-            $this->server .= substr($parsed['path'], 0, strripos($parsed['path'], '/'));
+            $this->server .= substr($parsed['path'], 0, strrpos($parsed['path'], '/'));
```

Comment:
Is there any reason to change from the case insentive version of the same function?

Reply:
Yes, the string that we are searching for is not case sensitive.

Reply:
Oh got it now, that's the path after the domain name. Sorry I misunderstood the piece of code